client/app/common/common-modal/CommonModalController.js

```diff
@@ -11,10 +11,11 @@ angular.module('EnvironmentManager.common').controller('CommonModalController',
     vm.action = configuration.action || 'Ok';
     vm.severity = (configuration.severity || 'Default').toLowerCase();
     vm.details = (configuration.details || []).map($sce.trustAsHtml);
-    vm.infoMode = configuration.infomode || false;
+    vm.infoMode = configuration.infomode;
```

Comment:
Is this still needed?

Reply:
This one, yes, it's the notification mode below that isn't required any more...